pkg/hostagent/hostagent.go

```diff
@@ -259,6 +265,50 @@ func (a *HostAgent) Run(ctx context.Context) error {
 		return err
 	}
 
+	if *a.y.Video.Display == "vnc" {
+		vncdisplay, vncoptions, _ := strings.Cut(*a.y.Video.VNC.Display, ",")
+		vnchost, vncnum, err := net.SplitHostPort(vncdisplay)
+		if err != nil {
+			return err
+		}
+		n, err := strconv.Atoi(vncnum)
+		if err != nil {
+			return err
+		}
+		vncport := strconv.Itoa(5900 + n)
+		vncpwdfile := filepath.Join(a.instDir, filenames.VNCPasswordFile)
+		vncpasswd, err := generatePassword(8)
```

Comment:
8 seems too short. What about 32?

Reply:
It only supports 8

Reply:
https://www.rfc-editor.org/rfc/rfc6143.html#section-7.2.2